orchagent/main.cpp

```diff
@@ -78,6 +78,8 @@ void usage()
     cout << "    -i INST_ID: set the ASIC instance_id in multi-asic platform" << endl;
     cout << "    -s: enable synchronous mode (depreacated, use -z)" << endl;
     cout << "    -z: redis communication mode (redis_async|redis_sync|zmq_sync), default: redis_async" << endl;
+    cout << "    -f swss_rec_filename: swss record log filename(default 'swss.rec')" << endl;
+    cout << "    -j sairedis_rec_filename:  sairedis record log filename(default sairedis.rec)" << endl;
```

Comment:
Remove the extra space after `sairedis_rec_filename:`?

Reply:
Fixed in latest commit